CriticalMass/Location.swift

```diff
@@ -7,7 +7,7 @@
 
 import Foundation
 
-struct Location: Equatable {
+public struct Location: Equatable, Hashable {
```

Comment:
Isn’t just Hashable enough since it implements Equatable I think

Reply:
Good point!